website/rss.py

```diff
@@ -0,0 +1,18 @@
+import feedparser
+import eventlet
+
+
+with eventlet.Timeout(10):
+    url = 'http://feeds.feedburner.com/PythonInsider/.rss'
+
+
+def get_rss():
+
+    feed = feedparser.parse(url)
+    posts_to_show = []
+
+    for post in feed.entries[0:4]:
```

Comment:
if a case where entries length is only 2 then this will break!

Reply:
I don't think so. You can check in shell.

Reply:
It will break

Reply:
I have checked first and then only I have replied.

Reply:
I am saying that feed.entries list can be empty or less than 3 some times, then feed.entries[:4] will fail!

Reply:
Check yourself first in shell.

>>> a = [10,20,30]
>>> print(a[:4])
[10, 20, 30]
>>> print(a[0:4])
[10, 20, 30]
>>> print(a[0:2])
[10, 20]
>>> print(a[0:0])
[]
>>> print(a[0:1])
[10]
>>> print(a[0:5])
[10, 20, 30]

Reply:
agreed